upgrade.py

```diff
@@ -465,7 +464,8 @@ def buildRestoreList(self, src_base):
         # CA-195388: Preserve /etc/mdadm.conf across upgrades
         restore_list += ['etc/mdadm.conf']
 
-        restore_list += ['var/lib/xcp/verify_certificates']
+        # CA-413099: Preserve /var/xapi/verify-certificates for certificates checking enable/disable status
```

Comment:
`/var/lib/xcp/verify-certificates` ?

Reply:
they actually refer to the same file, as symbol link as my commit message.
but good to update to avoid confusing.